specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/textanalytics.json

```diff
@@ -0,0 +1,2851 @@
+        "description": "Submit a collection of text documents for analysis.  Specify a single unique task to be executed immediately.",
+        "operationId": "AnalyzeText",
+        "consumes": [
+          "application/json",
+          "text/json"
+        ],
+        "produces": [
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ShowStats"
+          },
+          {
+            "description": "Collection of documents to analyze and a single task to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTask"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "A successful call result",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextTaskResult"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Entity Linking Request": {
+            "$ref": "./examples/SuccessfulEntityLinkingRequest.json"
+          },
+          "Successful Entity Recognition Request": {
+            "$ref": "./examples/SuccessfulEntityRecognitionRequest.json"
+          },
+          "Successful Key Phrase Extraction Request": {
+            "$ref": "./examples/SuccessfulKeyPhraseExtractionRequest.json"
+          },
+          "Successful PII Entity Recognition Request": {
+            "$ref": "./examples/SuccessfulPiiEntityRecognitionRequest.json"
+          },
+          "Successful Language Detection Request": {
+            "$ref": "./examples/SuccessfulLanguageDetectionRequest.json"
+          },
+          "Successful Sentiment Analysis Request": {
+            "$ref": "./examples/SuccessfulSentimentAnalysisRequest.json"
+          }
+        }
+      }
+    },
+    "/analyze-text/jobs": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "description": "Submit a collection of text documents for analysis. Specify one or more unique tasks to be executed as a long-running operation.",
+        "operationId": "AnalyzeText_SubmitJob",
+        "summary": "Submit text analysis job",
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "description": "Collection of documents to analyze and one or more tasks to execute.",
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeTextJobsInput"
+            }
```

Comment:
Here's another body parameter that is not marked required but the schema has required properties.  Is it really possible to get a successful response from this operation if no body is passed?

Reply:
Updated!